src/components/Person.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ name, age, sex, isMarried, partnerName }) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      <p className="Person__age">
+        {age ? `I am ${age}` : null}
+      </p>
+
+      <p className="Person__partner">
+        {isMarried
+          ? `My ${sex === 'm'
+            ? 'wife\'s'
+            : 'husband\'s'} name is ${partnerName}`
+          : 'I am not married'}
```

Comment:
nice

Reply:
Not nice.
Move complex logic from JSX to variable and NEVER use nested ternary operators.